lib/reline.rb

```diff
@@ -36,7 +36,7 @@ class Core
     attr_accessor :config
     attr_accessor :key_stroke
     attr_accessor :line_editor
-    attr_accessor :ambiguous_width
+    attr_writer :ambiguous_width
```

Comment:
It looks like "writer" for the `ambiguous_width` wasn't used originally, so while you're at it, would you delete this line?